content/operations/GOCDB/Accessing/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Accessing"
+description: "Accessing GOCDB"
+weight: 20
+type: "docs"
+---
+
+The GOCDB can be accessed at https://goc.egi.eu.
+
+To access the web interface, users can:
+
+- use an [X509 digital certificate installed in the browser](#using-an-x509-digital-certificate), delivered by
+  one of the recognised EU-Grid-PMA Certification Authorities ;
+- use EGI Check-in to [access the GOCDB with an institutional account](#using-institutional-account-via-egi-check-in).
+
+Users can access the system as soon as they are authenticated however they will
+only be able to update information based on their roles.
+More information about roles and associated permission is available in the
+[Users and
+roles](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Users_and_roles)
+section.
+
+All roles applications need to be validated by parent roles or administrators.
+Once this is done, you can access/modify relevant information according to the
+role you have been granted. You can learn more on roles and user accounts by
+reading the #Users and roles section of this documentation.
+
+## Using an X509 digital certificate
```

Comment:
Should we prioritise Check-in here? I would put it first

Reply:
I thought about ti too and was a bit unsure about this, as currently we are doing the opposite and the COU is a last resort.
Also ideally the X509 access should be done via Check-in IGTF IdP proxy, but while it works it will map people to a different user in GOCDB (if they already authentified via X509 to GOCDB)

Reply:
Implemented.